src/cgroups/cpu.rs

```diff
@@ -0,0 +1,88 @@
+//!
+//! Reference: https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#cpu
+
+use crate::{
+    cgroups::Controller,
+    rootfs::default_devices,
+    spec::{LinuxCpu, LinuxDeviceType, LinuxResources},
+};
+
+use anyhow::Result;
+use nix::unistd::Pid;
+use std::path::Path;
+
+pub struct Cpu {}
+
+impl Controller for Cpu {
+    fn apply(linux_resources: &LinuxResources, cgroup_root: &Path, pid: Pid) -> Result<()> {
+        fs::create_dir_all(cgroup_root)?;
```

Comment:
CI is failing to build this. You're using `fs::...` but you never imported it `use std::fs;`.